src/routes/reference/bufnmf.svx

```diff
@@ -9,4 +9,65 @@ tags:
     - spectral
 flair: reference
 category: Decompose
----
\ No newline at end of file
+---
+
+<script>
+    import IndentNote from '$lib/components/IndentNote.svelte';
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+</script>
+
+## BufNMF
+
+BufNMF uses Non-Negative Matrix Factorisation (NMF) to decompose the magnitudes of a spectrogram into a number of components specified by the user<sup>1</sup>. Each component is represented by the combination of two elements: a spectral template (“basis”, plural: “bases”) and an amplitude envelope (“activation”). Activations and bases can be used in various ways, including to resynthesize the audio of a single decomposed component. NMF is a popular technique in signal processing research for things like source separation and transcription<sup>2</sup>.
+
+<IndentNote type='warn'>
```

Comment:
I think should be type='pointer' for when you note to someone there is a related resource inline